kani-driver/src/args/mod.rs

```diff
@@ -242,6 +242,9 @@ pub struct VerificationArgs {
     #[arg(long)]
     pub default_unwind: Option<u32>,
 
+    #[arg(long = "export-json")]
```

Comment:
Just do:
```suggestion
    #[arg(long)]
```
as underscores are automatically mapped to hyphens.